src/backend/utils/inactive-blog-filter.js

```diff
@@ -38,17 +38,14 @@ function dateDiff(postDate) {
  */
 async function check(feedUrl, redlistPath = 'feeds-redlist.json') {
   // Read redlist file
-  return fsPromises
-    .readFile(redlistPath, 'utf-8')
-    .then(redListRaw => {
-      // Concat to array no matter what
-      const redList = [].concat(JSON.parse(redListRaw));
-      return Promise.resolve(redList.some(isRedlisted.bind(null, feedUrl)));
-    })
-    .catch(error => {
-      log.error({ error }, `failed to read ${redlistPath}`);
-      return Promise.reject(error);
-    });
+  try {
+    const redListRaw = await fsPromises.readFile(redlistPath, 'utf-8');
+    const redList = [].concat(JSON.parse(redListRaw));
+    return redList.some(isRedlisted.bind(null, feedUrl));
```

Comment:
Could this not be `return redList.some(redItem => isRedlisted(feedUrl, redItem));`?  It doesn't seem like we need to `.bind()` here.